src/Akka.Streams.Kafka.Testkit/MemberAssignment.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using Confluent.Kafka;
+
+namespace Akka.Streams.Kafka.Testkit
+{
+    // TODO: Need testing, check if little endian/big endianness is correct
+    /// <summary>
+    /// Specialized deserializer for MemberAssignment.
+    /// C# AdminClient returns raw bytes that needs to be deserialized into a proper object
+    /// </summary>
+    public class MemberAssignment
```

Comment:
The dotnet Kafka admin client does not deserialize MemberAssignment struct, need to make a custom deserializer for it.